lib/src/implementations/prismic_io_article_repository.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+import 'package:rxdart/rxdart.dart';
+import '../blocs/article_gettable.dart';
+import './prismic_io_article.dart';
+
+class PrismicIoArticleRepository implements ArticleGettable {
+  @override
+  ValueObservable<List<Article>> getArticles() {
```

Comment:
This function looks fetching articles only once after it's called. It's supposed to be `Future` instead